plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -233,7 +232,7 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
 	onReceiveOverrideFn := func(e v2.Event, d *channel.DataChan) error {
 		log.Info("got status check call,fire events for above publisher")
 		if len(eventManager.Stats) == 0 {
-			eventManager.PublishEvent(event.FREERUN, 0, "ptp-not-set", "PTP_STATUS")
+			eventManager.PublishEvent(ptp.FREERUN, 0, "ptp-not-set", "PTP_STATUS")
```

Comment:
instead of PTP_STATUS  EventPtpStateChange

Reply:
done